packages/core/package.json

```diff
@@ -14,7 +14,8 @@
         "build"
     ],
     "scripts": {
-        "build": "yarn build:styles",
+        "build": "echo 'noop'",
```

Comment:
Let keep both as a build? I think it is nice that `yarn run build` builds in all packages.

Reply:
Then will build core twice on yarn build from root.